charts/xp-management/templates/service-swagger.yaml

```diff
@@ -15,6 +15,6 @@ spec:
       targetPort: {{ .Values.swaggerUi.service.internalPort }}
       protocol: TCP
   selector:
-    app: {{ template "management-svc.fullname" .}}
+    app: {{ template "management-svc.name" .}}
```

Comment:
Might be nice to define a helper function `management-svc.labels`, similar to what is done for Turing/MLP/Merlin, and reuse it everywhere. Eg: https://github.com/caraml-dev/helm-charts/blob/main/charts/mlp/templates/_helpers.tpl

Reply:
We actually do, just that we didn't add the `app` and `release` labels to the helper function: https://github.com/caraml-dev/helm-charts/blob/main/charts/xp-management/templates/_helpers.tpl#L45

Reply:
No, I don't mean to use it for the selector. I missed that there is already a `management-svc.labels` helper function and we are including it for the pod labels too: https://github.com/caraml-dev/helm-charts/blob/main/charts/xp-management/templates/deployment.yaml#L25

Seems like the `release` label is also repeated. For this, shall we simply move the `app` label under the existing `management-svc.labels` and clean up the explicit `app` and `release` labels [here](https://github.com/caraml-dev/helm-charts/blob/main/charts/xp-management/templates/deployment.yaml#L23)?

Reply:
Oops okay I realise this thread and the one [above](https://github.com/caraml-dev/helm-charts/pull/353#discussion_r1314487263) are converging. 

Essentially only XP reuses the `management-svc.labels` template in its deployment's pod template, whereas all the other apps specifically mention the `app` and `release` labels in the deployment pod template (and do not reuse their `[app].labels` block). What this means is that XP's deployment pods have more labels than the other apps' pods (their pod labels are a subset of the deployment's labels).

Maybe a common standard for all the apps can be decided on another day but I think to make think things consistent in the short run, we can add the `app` and `release` labels to the `management-svc.labels` template (so the deployment contain these labels like the other CaraML apps), and at the same time remove this line https://github.com/caraml-dev/helm-charts/blob/main/charts/xp-management/templates/deployment.yaml#L25 (so that the pod template is similar to what we have also for the other CaraML apps).

Reply:
Ok, that sounds good to me.